Extension/browser/firefox/lib/contentScripts.js

```diff
@@ -194,6 +198,27 @@
     }
 
     /**
+     * Register css in user sheet
+     * @param cssContent Css content
+     */
+    function registerCss(cssContent) {
```

Comment:
No, we can't. Chrome and Safari allow to register css via manifest file, but Firefox doesn't. We need to register assistant's selector style.